src/cloud-api-adaptor/test/e2e/docker_test.go

```diff
@@ -129,3 +129,43 @@ func TestDockerCreatePeerPodWithAuthenticatedImageWithValidCredentials(t *testin
 		t.Skip("Registry Credentials, or authenticated image name not exported")
 	}
 }
+
+func TestDockerCreateWithCpuLimit(t *testing.T) {
+	t.Parallel()
+
+	assert := DockerAssert{}
+	DoTestPodWithCpuMemLimitsAndRequests(t, testEnv, assert, "", "", "1", "")
+}
+
+func TestDockerCreateWithCpuFractionalLimit(t *testing.T) {
+	SkipTestOnCI(t)
```

Comment:
Are these skipped on the CI for any particular reason?

Reply:
I was not sure how many tests you would want to run in the CI. Hence disabled it.

Reply:
Good point - I guess it is covered by unit tests at the moment, so low priority for a full e2e test?

Reply:
So I think it's fine for now and we see how long pod provisioning takes on the docker CI - if it's very quick we can consider overlapping coverage, otherwise if it's covered by unit tests we could consider removing some of the overlap e2e tests

Reply:
Can you attached the output of running the docker tests locally to give an indication of the time it takes please?

Reply:
What is the delta caused by each of the cpu/mem limit/request tests?

Reply:
Approx 75-86 s
Example.
```
=== RUN   TestDockerCreateWithMemLimit
=== RUN   TestDockerCreateWithMemLimit/PodWithCpuMemLimitsAndRequests_test
    assessment_runner.go:264: Waiting for containers in pod: pod-with-cpu-mem-limits-requests are ready
=== RUN   TestDockerCreateWithMemLimit/PodWithCpuMemLimitsAndRequests_test/Pod_with_cpu_and_memory_limits_and_requests
=== NAME  TestDockerCreateWithMemLimit/PodWithCpuMemLimitsAndRequests_test
    assessment_runner.go:623: Deleting pod pod-with-cpu-mem-limits-requests...
    assessment_runner.go:630: Pod pod-with-cpu-mem-limits-requests has been successfully deleted within 60s
--- PASS: TestDockerCreateWithMemLimit (20.10s)
    --- PASS: TestDockerCreateWithMemLimit/PodWithCpuMemLimitsAndRequests_test (20.09s)
        --- PASS: TestDockerCreateWithMemLimit/PodWithCpuMemLimitsAndRequests_test/Pod_with_cpu_and_memory_limits_and_requests (0.01s)
PASS
ok      github.com/confidential-containers/cloud-api-adaptor/src/cloud-api-adaptor/test/e2e     86.051s
```